src/libstrongswan/plugins/openssl/openssl_kem.c

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (C) 2024 Amazon.com Inc. or its affiliates.
```

Comment:
We only have your individual CLA on file (no entity CLA for Amazon), so I feel it would be better if you used your own name here.

Reply:
Thankfully, they all agree that we should align with the project's requirements/preferences. 👍